authors/apps/articles/serializers.py

```diff
@@ -59,3 +59,25 @@ def get_read_time(self,obj):
       
     def get_share_links(self,obj):
         return share_links_generator(obj,self.context['request'])
+
+
+class BookmarksSerializer(serializers.ModelSerializer):
+
+    title = serializers.SerializerMethodField()
+    slug =  serializers.SerializerMethodField()
+    description = serializers.SerializerMethodField()
+    author = AuthorProfileSerializer(read_only=True)
+
+    class Meta:
+        model = Bookmark
+        fields = ('title', 'slug', 'description','author')
+
```

Comment:
When a class is nested in a class, the next attribute that comes after it can be separated by one line. So, in this case, the second line is not required as both the function and the `Meta` class belong to the `BookmarksSerializer` class.